include/tvm/te/operation.h

```diff
@@ -112,6 +113,7 @@ class OperationNode : public tir::FunctionBaseNode {
       const Operation& self,
       arith::Analyzer* analyzer,
       const std::unordered_map<const VarNode*, IntSet>& dom_map,
+      const std::unordered_map<IterVar, Range>& rmap,
```

Comment:
Given that we already pass the analyzer around, shall we simply bind the same of each variabe to the analyzer?

Reply:
Don't need this change as no need to pass rmap to EvalSet any more.